frontend/src/app/subject-comments/subject-comments.component.css

```diff
@@ -0,0 +1,7 @@
+.happy{
```

Comment:
Happy and sad aren't descriptive enough. flagged and unflagged should be the messages. Let's combine them with the btn tag, so it's explicit that it applies to buttons only.

So it should be somthing like `.btn, .unflagged { ... }` and `.btn, .flagged { ... }`.

Reply:
oo Ic, I was using them to provide an overall description for both flagged/unflagged and thumbup/thumbdown.

I've fixed this so that all 4 have separate classes and are more descriptive! :dancer: